ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -129,6 +129,7 @@ tests:
         cluster_profile=/var/run/secrets/ci.openshift.io/cluster-profile
         export AWS_SHARED_CREDENTIALS_FILE=${cluster_profile}/.awscred
         export KUBE_SSH_KEY_PATH=${cluster_profile}/ssh-privatekey
+        export WINDOWS_SERVER_VERSION="2019"
```

Comment:
Any reason to not use
```
    env:
      WINDOWS_SERVER_VERSION: "2019"
```

Reply:
I would have to add `WINDOWS_SERVER_VERSION` to [optional-operators-ci-aws-ovn-hybrid-workflow.yaml](https://github.com/openshift/release/blob/master/ci-operator/step-registry/optional-operators/ci/aws/ovn/hybrid/optional-operators-ci-aws-ovn-hybrid-workflow.yaml) similar to [windows-operator-e2e-test-ref.yaml](https://github.com/openshift/release/blob/a0b0c38550c7529c216097565340055c9f0c52aa/ci-operator/step-registry/windows/e2e/operator/test/windows-e2e-operator-test-ref.yaml#L14). We shouldn't be adding something Windows specific to a generic job.